dashboard/app/models/pd/workshop.rb

```diff
@@ -587,6 +587,10 @@ def csf?
     course == COURSE_CSF
   end
 
+  def csf_201?
```

Comment:
Can we get a comment explaining what 201 means?

Reply:
Do you mean explaining what SUBJECT_CSF_201 is? It's explained in lib/cdo/shared_constants/pd/shared_workshop_constants.rb  `SUBJECT_CSF_201 = 'Deep Dive'.freeze`